notebooks/106-auto-device/106-auto-device.ipynb

```diff
@@ -0,0 +1,485 @@
+    "# Initilization for inference with THROUGHPUT hint\n",
+    "context.reset()\n",
+    "\n",
+    "job_id = 0\n",
+    "while context.feed_inference:\n",
+    "    infer_queue.start_async({input_layer_ir.any_name: input_image}, InferJob(job_id, context))\n",
+    "    period_exec_time = time.perf_counter() - context.period_start_time\n",
+    "    job_id += 1\n",
+    "    \n",
+    "infer_queue.wait_all()\n",
+    "\n",
+    "# Take the fps and latency of latest period\n",
+    "THROUGHPUT_fps = context.period_fps\n",
+    "THROUGHPUT_latency = context.period_latency\n",
+    "\n",
+    "print(\"Done\")\n",
+    "sys.stdout.flush()\n",
+    "# print(overall_latency_list)\n",
+    "del compiled_model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a63d446b-0c7c-4f2d-baac-21d5ea92089d",
+   "metadata": {},
+   "source": [
+    "### Inference with LATENCY hint\n",
+    "\n",
+    "Loop for the inference and output the FPS/Latency for each @period_seconds sencods"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "42e0396c-d7cb-4176-844c-1d5d94ae788f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Compiling Model for AUTO Device with LATENCY hint\")\n",
+    "sys.stdout.flush()\n",
+    "\n",
+    "compiled_model = ie.compile_model(model=model, device_name=\"AUTO\", config={\"PERFORMANCE_HINT\":\"LATENCY\"})\n",
+    "\n",
+    "infer_queue = AsyncInferQueue(compiled_model, 0)  #set 0 will query optimal num by default\n",
+    "infer_queue.set_callback(completion_callback)\n",
+    "\n",
+    "print(f\"Start inference, {end_after_periods:.0f} groups fps/latency will be out with {period_seconds:.0f}s interval\")\n",
+    "sys.stdout.flush()\n",
+    "\n",
+    "# Initilization for inference with LATENCY hint\n",
+    "context.reset()\n",
+    "\n",
+    "job_id = 0\n",
+    "while context.feed_inference:\n",
+    "    infer_queue.start_async({input_layer_ir.any_name: input_image}, InferJob(job_id, context))\n",
+    "    period_exec_time = time.perf_counter() - context.period_start_time\n",
+    "    job_id += 1\n",
+    "    \n",
+    "infer_queue.wait_all()\n",
+    "\n",
+    "# Take the fps and latency of latest period\n",
+    "LATENCY_fps = context.period_fps\n",
+    "LATENCY_latency = context.period_latency\n",
+    "\n",
+    "print(\"Done\")\n",
+    "sys.stdout.flush()\n",
+    "# print(overall_latency_list)\n",
+    "del compiled_model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9f1907e4-e71b-471f-891a-66161c03bc85",
+   "metadata": {},
+   "source": [
+    "### FPS and latency difference\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "86f5261e-581e-4b91-a8fb-9715981d450e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# output the difference\n",
+    "labels = [\"fps\", \"latency\"]\n",
+    "THROUGHPUT = [THROUGHPUT_fps, THROUGHPUT_latency]\n",
+    "LATENCY = [LATENCY_fps, LATENCY_latency]\n",
```

Comment:
![image](https://user-images.githubusercontent.com/10940214/161101800-ffbbfd53-2570-429a-8b80-1aae1ecb8e13.png)
I know what you want to show but, the double-cross relation is not working to understand the idea of this graph.

Reply:
Maybe add MODE to THROUGHPUT and LATENCY legend values.

Reply:
![image](https://user-images.githubusercontent.com/19401820/161422209-aec72878-f9f3-456e-a36b-b5ac308f4380.png)